CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required (VERSION 2.6)
+project(ale)
+
+option(USE_SDL "Use SDL" OFF)
+option(USE_RLGLUE "Use RL-Glue" OFF)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-multichar -Wunused -fno-rtti -fPIC -O3 -fomit-frame-pointer")
```

Comment:
These were just the flags used by the unix makefile. Please remove if you
want to
On Feb 21, 2015 12:23 PM, "Marc G. Bellemare" <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/34#discussion_r25124267>
> :
>
> > @@ -0,0 +1,60 @@
> > +cmake_minimum_required (VERSION 2.6)
> > +project(ale)
> > +
> > +option(USE_SDL "Use SDL" OFF)
> > +option(USE_RLGLUE "Use RL-Glue" OFF)
> > +
> > +set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-multichar -Wunused -fno-rtti -fPIC -O3 -fomit-frame-pointer")
>
> Do we need no-rtti? The internet suggests this is only useful to save a
> bit of space, and it kills dynamic_cast, which on the long run could cause
> someone headaches.
>
> Also (and somewhat pedantically), why -Wno-multichar?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/34/files#r25124267>
> .
>